### PR TITLE
Create UserRegistration route

### PR DESCRIPTION
**Pull request by Genza999**
**What does this PR do?**
Creates a view function for registering User

**Description of Task to be completed?**
View function was created to help register users onto the application using Django's default User table. User registers with their email, username and password. Test the URL route using Postman. If it's successful in registering a User, a 201 HTTP response is returned with a JSON body of the user credentials, it is not successful, a 400 HTTP response with an error message is returned.

**Any background context you want to provide?**
Before you run the application, please install all the dependencies in the requirements.txt file.

**What are the relevant pivotal tracker stories?**
#164778316

Please review 
@anyatibrian @patrickf949 @e-ian

### CODE REVIEW

**Comment by patrickf949** [comment]
thumbs up @Genza999

File: Authentication/user/views.py [hunk 1]
```diff
@@ -1,3 +1,33 @@
 from django.shortcuts import render
```
Comment (e-ian) [inline Authentication/user/views.py:1]: @Genza999 Please include serializer file

Comment (anyatibrian) [reply Authentication/user/views.py:1]: great work

**Review by anyatibrian — APPROVED** [review]

File: Authentication/user/views.py (hunk 1 above)
Comment (anyatibrian) [reply Authentication/user/views.py:1]: great work

**Review by anyatibrian — APPROVED** [review]

**Pull request merged**